sensor_msgs/Image image
---
iri_perception_msgs/GeoVwSet geo_vw_set
